tests/swap_test.go

```diff
@@ -298,3 +294,7 @@ func getSwapFreeSizeInKib(node v1.Node) (size int) {
 func getSwapSizeInKib(node v1.Node) (size int) {
 	return getMemInfoByString(node, "SwapTotal")
 }
+
+func runVMIAndExpectLaunch(vmi *virtv1.VirtualMachineInstance, timeout int) *virtv1.VirtualMachineInstance {
+	return tests.RunVMIAndExpectLaunch(vmi, timeout)
+}
```

Comment:
What is the added value of this function? Can't we just replace `runVMIAndExpectLaunch` with `tests.RunVMIAndExpectLaunch`?

Reply:
+1

Reply:
DONE. thanks!